peekingduck/nodes/model/posenet.py

```diff
@@ -59,6 +60,10 @@ class Node(AbstractNode):
         score_threshold (:obj:`float`): **[0, 1], default = 0.4**. |br|
             Detected keypoints confidence score threshold, only keypoints above
             threshold will be kept in output.
+        use_jit (:obj:`bool`): **default = True**. |br|
```

Comment:
Likewise, this should be `default = False`.

Reply:
Changed